Dockerfile

```diff
@@ -5,10 +5,6 @@ RUN apt update
 RUN apt-get install wget
 RUN apt-get install less
 RUN apt-get install nano
-RUN wget -qO- https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/5.2.4/flyway-commandline-5.2.4-linux-x64.tar.gz | tar xvz \
- && ln -s `pwd`/flyway-5.2.4/flyway /usr/local/bin
-
-COPY src/metalnx-tools/src/main/resources/migrations/ /migrations
```

Comment:
Any idea what flyway is used for?

Reply:
flyway was used for db migrations

Reply:
Thanks. I eventually came across the docs which stated that. Just wanted to be sure this didn't affect people's ability to customize the look of the application.